src/AdminConsole/Pages/App/Settings/Settings.cshtml.cs

```diff
@@ -54,6 +57,8 @@ public class SettingsModel : BaseExtendedPageModel
 
     public ApiKeysModel ApiKeysModel { get; }
 
+    public bool IsGenerateSignInTokenEndpointEnabled { get; private set; }
```

Comment:
Not sure what's the best way to call it, but maybe we should stay consistent with the recently adopted naming of "manually generated verification tokens" in code too.

```suggestion
    public bool IsManuallyGeneratedTokensEnabled { get; private set; }
```